config/electronBuilder.config.js

```diff
@@ -1,23 +1,28 @@
 const {version} = require('../package.json');
 
 const isPublishing = process.argv.includes('--publish');
+const pullRequestNumber = process.env.PULL_REQUEST_NUMBER;
 
 const s3Bucket = {
     production: 'expensify-cash',
     staging: 'staging-expensify-cash',
     internal: 'ad-hoc-expensify-cash',
 };
 
+const s3Path = {
+    production: '/',
+    staging: '/',
+    internal: process.env.PULL_REQUEST_NUMBER
+        ? `/desktop/${pullRequestNumber}/`
+        : '/',
+};
+
 const macIcon = {
     production: './desktop/icon.png',
     staging: './desktop/icon-stg.png',
     internal: './desktop/icon-stg.png',
 };
 
-const isCorrectElectronEnv = ['production', 'staging', 'internal'].includes(
```

Comment:
NAB but let's actually throw an error if we find that `process.env.ELECTRON_ENV` is not on of these?